ceph/UI/helper.py

```diff
@@ -0,0 +1,23 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+def take_ss(driver, filename: str):
+    """Takes a screenshot and saves it to the specified filename."""
+    try:
+        driver.save_screenshot(filename)
```

Comment:
Add logic on returning the screenshot path. The user should be able to get the file name upon calling this method